src/components/ContactCard/ContactFormField.jsx

```diff
@@ -65,8 +82,8 @@ ContactFieldInput.propTypes = {
   type: PropTypes.string.isRequired,
   placeholder: PropTypes.string,
   withLabel: PropTypes.bool,
-  required: PropTypes.bool,
-  labelPlaceholder: PropTypes.string
+  labelPlaceholder: PropTypes.string,
+  required: PropTypes.bool
```

Comment:
[Meta] When you review your changes, it's best if you can revert such unneeded changes before commit.

Reply:
+1. Using git add -p helps here. If you are on mac, gitup is a very simple GUI to stage hunks and reverting hunks.